### ISSUE TITLE
get rid of `total_netted` in `NettingChannelLibrary.sol`

### ISSUE THREAD

**Issue by czepluch** [raiden-network/raiden#362]
It seems we're not using `total_netted` at any point in the smart contract. We assign it here (https://github.com/raiden-network/raiden/blob/master/raiden/smart_contracts/NettingChannelLibrary.sol#L405), but we never use it from there.

**Comment by hackaugusto** [comment]
That variable was used in a check like:

```
       if (total_netted != total_deposit) {
           throw;
```

Turns out this was a bug, fixed by commit 5490633f162b985862c080a00ae7530b0234c32c , here is the reasoning as explained in the commit message:

```
   - `settle()` now pays out first to the non-closing party and only
      the amount that it can afford. Afterwards if anything is left it is
      paid out to the party that closed the channel.
```

So, it's safe to remote it.

**Comment by czepluch** [comment]
closed with #364
